BASE/README.md

```diff
@@ -0,0 +1,121 @@
+[![Build Status](https://mseng.visualstudio.com/AppInsights/_apis/build/status/ChuckNorris/AI-BaseSDK-develop-linux?branchName=develop)](https://mseng.visualstudio.com/AppInsights/_build/latest?definitionId=6237?branchName=develop)
```

Comment:
why is this appearing as a new file instead of simple renaming?

Reply:
i did copy the readme. i wanted to make sure i preserve a copy in each sub folder, but i still needed one in the root for github to display